update.sh

```diff
@@ -61,5 +64,4 @@ fi
 
 # Allow the user to know that the upgrade has completed.
 echo "---
-The upgrade has been completed. Please reboot your system to see the changes.
----"
\ No newline at end of file
+The upgrade has been completed. Please reboot your system to see the changes.---"
```

Comment:
Why is this changed?

Reply:
I am unsure. I'll review it in further detail when I am home.